packages/boxel/addon/components/boxel/tab-bar/index.hbs

```diff
@@ -0,0 +1,23 @@
+<nav role="tablist" class={{cn "boxel-tab-bar" @class}} ...attributes>
+  {{#if @items}}
+    {{#each (compact @items) as |tab|}}
+      <div
+        role="none"
+        class={{cn
+          "boxel-tab-bar__item"
+          boxel-tab-bar__item--is-active=tab.action.isActive
+        }}
+      >
+        {{!-- template-lint-disable require-context-role --}}
+        <a
```

Comment:
Why did we choose `a` over `button` for this? If we do `event.preventDefault` and the `href` is not representative of triggered navigation,  it seems like we could use `button` instead.

Reply:
True! I just copied this from `Menu`, I’m not sure why it uses `a`. BUT I am now convinced I want these to be true links, I always resent when I know something is link-like and developer decisions have made me unable to use it like one 😬

Reply:
We still can't open things in a new tab if we're preventing the default event and navigating with Javascript. We might need different HTML + event handling for actions and navigation?